baseplate/lib/experiments/__init__.py

```diff
@@ -183,7 +186,13 @@ def variant(
 
         :return: Variant name if a variant is active, None otherwise.
```

Comment:
might be nice to add something like:

```rst
.. versionchanged:: 1.5
    ``name`` was renamed to ``experiment_name``
```

so it's clear in the docs when this happened